src/App.tsx

```diff
@@ -15,49 +17,121 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
```

Comment:
nit: Consider using an interface instead of a type alias for object types, as it is more flexible and easier to read. Change `type ReorderOptions` to `interface ReorderOptions`.